cli/src/cmd_net.rs

```diff
@@ -1781,14 +1781,11 @@ async fn create_current(settings_id: Uuid, client: &Client) -> Result<SwitchPort
         routes: current
             .routes
             .iter()
-            .map(|x| {
-                let gw: oxnet::IpNet = x.gw.to_string().parse().unwrap();
-                Route {
-                    dst: x.dst.clone(),
-                    gw: gw.addr(),
-                    vid: x.vlan_id,
-                    rib_priority: x.rib_priority,
-                }
+            .map(|x| Route {
+                dst: x.dst.clone(),
+                gw: x.gw,
+                vid: x.vlan_id,
+                rib_priority: x.rib_priority,
```

Comment:
Consider deleting `iter()` and

```suggestion
            .into_iter()
            .map(
                |SwitchPortRouteConfig {
                     dst,
                     gw,
                     rib_priority,
                     vlan_id,
                     ..
                 }| Route {
                    dst,
                    gw,
                    vid: vlan_id,
                    rib_priority,
                },
            )
```

Or maybe just `into_iter` and delete the `clone`

You might take a look at API consistency: why is it `vid` vs. `vlan_id`?